include/lmms_math.h

```diff
@@ -280,6 +284,53 @@ class LinearMap
 	T m_b;
 };
 
+#ifdef __SSE2__
+// exp approximation for SSE2: https://stackoverflow.com/a/47025627/5759631
+// Maximum relative error of 1.72863156e-3 on [-87.33654, 88.72283]
+static inline __m128 fast_exp_sse(__m128 x)
```

Comment:
```suggestion
inline __m128 fastExp(__m128 x)
```
The `static` isn't needed, and the name could be changed to fit the style guidelines.

Since the parameter takes an SSE type, I don't think it's necessary to include "SSE" in the function name.